### ISSUE TITLE
[rtrace] "passed successfully" on failed test

### ISSUE THREAD

**Issue by trusktr** [jtenner/as-pect#84]
I'm seeing some output like:

```
✔ src/as/glas/core/Layers.spec.ts Pass: 5 / 5 Todo: 0 Time: 1.632ms
❌ src/as/glas/math/Color.spec.ts Pass: 4 / 5 Todo: 0 Time: 1.741ms
  Color.setHSL
  Reason: Test suite Color.setHSL passed successfully.    ❌ sets color from HSL values - 
      [Expected]: 0.75
      [Actual]  : 0.833333333333333
```

Note the `Test suite Color.setHSL passed successfully` there, although that test failed.

**Comment by trusktr** [comment]
Reproduction:

```sh
git clone git@github.com:infamous/glas.git
cd glas
git checkout as-pect-issue-84
npm i
npm test
```

**Comment by jtenner** [comment]
Yep! I wanted to make sure that I caught every small thing. It's just hard to see everything that goes wrong when the tests fail. That's what the jest tests are for.

I plan to address this problem with more precision in 1.2.0

**Comment by jtenner** [comment]
Looks like this is a formatting issue, and a message issue.

I will fix this in `1.1.1` and refactor the jest test suite properly in `1.2.0`.
